src/de/jost_net/JVerein/server/MitgliedImpl.java

```diff
@@ -1297,16 +1298,703 @@ private Date toDate(String datum)
+
+    private final static String TITEL = "Dr. Dr.";
+
+    private final static String NAME = "Wichtig";
+
+    private final static String VORNAME = "Willi";
+
+    private final static String STRASSE = "Bahnhofstr. 22";
+
+    private final static String ADRESSZUSATZ = "Hinterhof bei M�ller";
+
+    private final static String PLZ = "12345";
+
+    private final static String ORT = "Testenhausen";
+
+    private final static String STAAT = "Deutschland";
+
+    private final static String BIC = "XXXXXXXXXXX";
+
+    private final static String IBAN = "DE89370400440532013000";
+
+    private final static Integer ZAHLUNGSWEG = 1;
+
+    private final static Date MANDATDATUM = new Date();
+
+    private final static String MANDATID = "12345";
+
+    private final static Date LETZTE_LASTSCHRIFT = new Date();
+
+    private final static String EXTERNE_MITGLIEDSNUMMER = "123456";
+
+    private final static String ID = "1";
+
+    private final static Zahlungsrhythmus ZAHLUNGSRHYTHMUS = new Zahlungsrhythmus(
+        12);
+
+    private final static Zahlungstermin ZAHLUNGSTERMIN = Zahlungstermin.HALBJAEHRLICH4;
+
+    private final static Integer MANDATVERSION = 1;
+
+    private final static String MAIL = "willi.wichtig@jverein.de";
+
+    private final static String GESCHLECHT = GeschlechtInput.MAENNLICH;
+
+    private final static Date GEBURTSDATUM = toDate("02.03.1980");
+
+    private final static String TELEFON_PRIVAT = "011/123456";
+
+    private final static String TELEFON_DIENSTLICH = "011/123456789";
+
+    private final static String HANDY = "0170/123456789";
+
+    private final static Date EINTRITT = toDate("01.01.2010");
+
+    private final static Beitragsgruppe BEITRAGSGRUPPE = beitragsgruppe();
+
+    private final static double INDIVIDUELLER_BEITRAG = 123.45;
+
+    private final static long ZAHLERID = 123456;
+
+    private final static Date AUSTRITT = toDate("01.01.2025");
+
+    private final static Date KUENDIGUNG = toDate("01.11.2024");
+
+    private final static Date STERBETAG = toDate(("31.12.2024"));
+
+    private final static String VERMERK1 = "Vermerk 1";
+
+    private final static String VERMERK2 = "Vermerk 2";
+
+    private final static Date EINGABEDATUM = toDate("01.02.2010");
+
+    private final static Date LETTZTE_AENDERUNG = new Date();
+
+    private final static boolean IS_ANGEMELDET = true;
+
+    private final static Map<String, String> VARIABLEN = new HashMap<>();
+
+    private final static Adresstyp ADRESSTYP = new Adresstyp()
+    {
+      @Override
+      public void transactionBegin() throws RemoteException
+      {
+
+      }
+
+      @Override
+      public void transactionCommit() throws RemoteException
+      {
+
+      }
+
+      @Override
+      public void transactionRollback() throws RemoteException
+      {
+
+      }
+
+      @Override
+      public String getBezeichnung() throws RemoteException
+      {
+        return "Adresstyp";
+      }
```

Comment:
Das hier sollte Mitglied heißen da unten die ID 1 zurückgeliefert wird.
![Bildschirmfoto_20241120_203248](https://github.com/user-attachments/assets/0a97f265-b384-4edb-bc65-cc8568740fdf)

Reply:
Vielleicht sollte man die Klasse wirklich mal umbenennen. Man kann ja die Datenbank Tabelle beim alten Namen lassen. Für neue JVerein Entwickler ist halt die Historie nicht immer bekannt.

Das gleiche gilt dann auch für Mitgliedskonto welches man in Sollbuchung umbenennen sollte.

Wenn wir mal einen Stand haben wo nicht zuviele gleichzeitig arbeiten könnte ich das mal machen. Es sind dann halt viele Files betroffen.